adserver/decisionengine/backends.py

```diff
@@ -112,6 +113,9 @@ def get_analyzer_keywords(self):
         if not self.url:
             return None
 
+        if "adserver.analyzer" not in settings.INSTALLED_APPS:
```

Comment:
Is it worth logging here? Might be a bit spammy tho.

Reply:
It would be *very* spammy, but if we log at debug it should be fine.